script.js

```diff
@@ -0,0 +1,27 @@
+function check(number) {
```

Comment:
`checkIsNumber`. Check is too generic to me

Reply:
done